messenger_users/views.py

```diff
@@ -79,6 +80,35 @@ def confirm_linkedin_profile(request):
 # def browse_saved_jobs(request):
 #     pass
 
+def search_linkedIn_profiles(request):
+    data = json.loads(request.body)
+    messenger_id = data['messenger_id']
+    username = data['linkedin_username']
+
+    searchObject = Search()
```

Comment:
for this to globals.py, this is a singleton object and we're only initializing one instance of it. Remember to import it from globals.py